Tagged release build is broken

**Describe the bug**
I tried doing an 11.0.1-M1 release and the build failed with staging rule failures on the whitebox artifacts:

```
[ERROR] Rule failure while trying to close staging repository with ID "jakartatck-1229".
[ERROR] 
[ERROR] Nexus Staging Rules Failure Report
[ERROR] ==================================
[ERROR] 
[ERROR] Repository "jakartatck-1229" failures
[ERROR]   Rule "pom-staging" failures
[ERROR]     * Invalid POM: /jakarta/tck/annotated/11.0.1-M1/annotated-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/whitebox-tx/11.0.1-M1/whitebox-tx-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/whitebox-notx-param/11.0.1-M1/whitebox-notx-param-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/whitebox-tx-param/11.0.1-M1/whitebox-tx-param-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/mixedmode/11.0.1-M1/mixedmode-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/whitebox-xa-param/11.0.1-M1/whitebox-xa-param-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/whitebox-notx/11.0.1-M1/whitebox-notx-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/whitebox/11.0.1-M1/whitebox-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/mdcomplete/11.0.1-M1/mdcomplete-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/ibanno/11.0.1-M1/ibanno-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/whitebox-xa/11.0.1-M1/whitebox-xa-11.0.1-M1.pom: Project name missing
[ERROR]     * Invalid POM: /jakarta/tck/multianno/11.0.1-M1/multianno-11.0.1-M1.pom: Project name missing
[ERROR] 
[ERROR] 
[ERROR] Cleaning up local stage directory after a Rule failure during close of staging repositories: [jakartatck-1229]
[ERROR]  * Deleting context 11702ee95a2c9c.properties
[ERROR] Cleaning up remote stage repositories after a Rule failure during close of staging repositories: [jakartatck-1229]
[ERROR]  * Dropping failed staging repository with ID "jakartatck-1229" (Rule failure during close of staging repositories: [jakartatck-1229]).
```

These need to be fixed to stage a milestone.

--- a/tcks/apis/connector-whitebox/annotated/pom.xml
+++ b/tcks/apis/connector-whitebox/annotated/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>annotated</artifactId>
+    <name>whitebox-anno_no_md</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/ibanno/pom.xml
+++ b/tcks/apis/connector-whitebox/ibanno/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>ibanno</artifactId>
+    <name>whitebox-ibanno_no_md</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/mdcomplete/pom.xml
+++ b/tcks/apis/connector-whitebox/mdcomplete/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>mdcomplete</artifactId>
+    <name>whitebox-mdcomplete</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/mixedmode/pom.xml
+++ b/tcks/apis/connector-whitebox/mixedmode/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>mixedmode</artifactId>
+    <name>whitebox-mixedmode</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/multianno/pom.xml
+++ b/tcks/apis/connector-whitebox/multianno/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>multianno</artifactId>
+    <name>whitebox-multianno</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/whitebox-notx-param/pom.xml
+++ b/tcks/apis/connector-whitebox/whitebox-notx-param/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>whitebox-notx-param</artifactId>
+    <name>whitebox-notx-param</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/whitebox-notx/pom.xml
+++ b/tcks/apis/connector-whitebox/whitebox-notx/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>whitebox-notx</artifactId>
+    <name>whitebox-notx</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/whitebox-tx-param/pom.xml
+++ b/tcks/apis/connector-whitebox/whitebox-tx-param/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>whitebox-tx-param</artifactId>
+    <name>whitebox-tx-param</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/whitebox-tx/pom.xml
+++ b/tcks/apis/connector-whitebox/whitebox-tx/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>whitebox-tx</artifactId>
+    <name>whitebox-tx</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/whitebox-xa-param/pom.xml
+++ b/tcks/apis/connector-whitebox/whitebox-xa-param/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>whitebox-xa-param</artifactId>
+    <name>whitebox-xa-param</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/whitebox-xa/pom.xml
+++ b/tcks/apis/connector-whitebox/whitebox-xa/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>whitebox-xa</artifactId>
+    <name>whitebox-xa</name>
     <packaging>rar</packaging>
 
     <properties>

--- a/tcks/apis/connector-whitebox/whitebox/pom.xml
+++ b/tcks/apis/connector-whitebox/whitebox/pom.xml
@@ -11,6 +11,7 @@
     </parent>
 
     <artifactId>whitebox</artifactId>
+    <name>whitebox-jar</name>
     <packaging>jar</packaging>
 
     <properties>

